todo_app/tests_e2e/app_e2e_test.py

```diff
@@ -0,0 +1,99 @@
+import pytest
+import todo_app.app as app
+import todo_app.data.session_items as session
+from threading import Thread
+import requests, os
+from selenium import webdriver
+from dotenv import load_dotenv, find_dotenv
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+
+@pytest.fixture(scope='module')
+def test_app():
+# Create the new board & update the board id environment variable    
+    file_path = find_dotenv('.env')
+    load_dotenv(file_path, override=True)
+    board_id = session.create_trello_board()
+    os.environ['TRELLO_BOARD_ID'] = board_id
```

Comment:
What do you imagine this line is doing?

Your app doesn't read TRELLO_BOARD_ID anywhere. It finds by board name. If you want to keep that, then you need to make the board name configurable.

It also looks for list by name, and doesn't use the default list names. So you either need to change that/make it configurable or set up the test board with the correct list names.

Currently your selenium test uses the real board.